sasmodels/convert.py

```diff
@@ -52,6 +52,12 @@
     ("_pd_n", ".npts"),
     ("_pd_nsigma", ".nsigmas"),
     ("_pd_type", ".type"),
```

Comment:
Yes, these should stay for the 3.x to 4.x conversions. I don't explicitly handle them because your work already has.